libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/SFLoginViewController.h

```diff
@@ -26,7 +26,24 @@
  WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-#import "SalesforceSDKCore.h"
```

Comment:
This caused the a ripple effect of needing to import header files in places they weren't, but the reality is that this class wasn't actually making use of the broad-brush header import, and that kind of thing tends to have unintended consequences over time.